py/objtype.c

```diff
@@ -533,6 +533,14 @@ STATIC void mp_obj_instance_load_attr(mp_obj_t self_in, qstr attr, mp_obj_t *des
 
     // try __getattr__
     if (attr != MP_QSTR___getattr__) {
+        #if MICROPY_PY_METHODS_DELATTRS_SETATTRS
+        // if the requested attr is __setattr__ assign MP_OBJECT_NULL to dest[0] to indicate success
+        if (attr == MP_QSTR___setattr__) {
+            dest[0] = MP_OBJ_NULL;
```

Comment:
@dmazzella : @dpgeorge talked about "line", not "lines". These fine distinctions are very important in programming.